definitions/tools/bcftools_view.cwl

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+
+baseCommand: ["/opt/bcftools/bin/bcftools", "view"]
+
+requirements:
+    - class: ResourceRequirement
+      ramMin: 4000
+    - class: DockerRequirement
+      dockerPull: "mgibio/bcftools-cwl:1.12"
+
+inputs:
+    sample_name:
+        type: string?
+        inputBinding:
+            position: 1
+            prefix: "--samples"
+        doc: "comma separated list of samples to include (or exclude with '^' prefix)"
+    output_type:
+        type:
+            type: enum
+            symbols: ["b", "u", "z", "v"]
+        default: "z"
+        inputBinding:
+            position: 4
+            prefix: "--output-type"
+        doc: "output file format"
```

Comment:
Is it worth mentioning what the options mean?

Reply:
done!